frontend/tests/components/admin/droplet-client.test.tsx

```
@@ -1,13 +1,33 @@
 jest.mock("@/components/admin/droplets/droplet-block", () => ({
   DropletBlock: ({ droplet }: { droplet: any }) => (
     <div data-testid={`droplet-${droplet.id}`}>{droplet.name}</div>
   ),
 }));
 
+// Mock lodash debounce to execute immediately in tests
+jest.mock("lodash", () => {
+  const actual = jest.requireActual("lodash");
+  return {
+    ...actual,
+    debounce: (fn: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)